api/proto/teleport/legacy/types/types.proto

```diff
@@ -971,6 +981,8 @@ message Rewrite {
   // Headers is a list of headers to inject when passing the request over
   // to the application.
   repeated Header Headers = 2 [(gogoproto.jsontag) = "headers,omitempty"];
+  // JWTClaims configures whether roles/traits are included in the JWT token.
+  JWTClaimsRewriteType JWTClaims = 3 [(gogoproto.jsontag) = "jwt_claims,omitempty"];
```

Comment:
You have a custom marshaler/unmarshaler for this below, this field can be defined as a string in a resource, correct? So users wouldn't have to write it as an integer?

Reply:
Yeah, it can be provided as a string